var/spack/repos/builtin/packages/verilator/package.py

```diff
@@ -87,7 +88,7 @@ class Verilator(AutotoolsPackage):
 
     conflicts("%gcc@:6", msg="C++14 support required")
 
-    patch("fix_compile_gch.patch", level=1, when="@5.0.18:")
+    patch("fix_compile_gch.patch", level=1, when="@5.0.18:5.028")
```

Comment:
The opening version in this range is formatted incorrectly. I suspect you mean:
```suggestion
    patch("fix_compile_gch.patch", level=1, when="@5.018:5.028")
```